### ISSUE TITLE
Problem when address of recipient begins with 0's

### ISSUE THREAD

**Issue by AntoineRondelet** [AntoineRondelet/snark-mixer#3]
I went through a case where the address of the recipient was: `0x00ec379776641a6200a4f930cd031948f9aeab6c`, and the withdraw function failed.

This is probably due to a error due to leading 0's being stripped from the binary string of the nullifier.

**Comment by HarryR** [comment]
Weird, does the same thing happen when using the `miximus` repository?

**Comment by barryWhiteHat** [comment]
Yeah that is really annoying. @AntoineRondelet I am so glad you are testing this stuff out :D
